ui/react-components/api/recurringAppointmentsApi.js

```diff
@@ -8,3 +8,12 @@ export const saveRecurringAppointments = async data => {
         console.error(error);
     }
 };
+
+export const getRecurringAppointmentByUuid = async (appointmentUuid) => {
```

Comment:
can we rename this to reucurringAppointmentFor??

Reply:
Done